cluster-autoscaler/core/scaledown/actuation/delete_in_batch.go

```diff
@@ -195,6 +195,9 @@ func CleanUpAndRecordFailedScaleDownEvent(ctx *context.AutoscalingContext, node
 		klog.Errorf("Scale-down: couldn't delete empty node, %v, status error: %v", errMsg, status.Err)
 		ctx.Recorder.Eventf(node, apiv1.EventTypeWarning, "ScaleDownFailed", "failed to delete empty node: %v", status.Err)
 	}
+	gpuConfig := ctx.CloudProvider.GetNodeGpuConfig(node)
+	metricResourceName, metricGpuType := gpu.GetGpuInfoForMetrics(gpuConfig, ctx.CloudProvider.GetAvailableGPUTypes(), node, nil)
+	metrics.RegisterFailedScaleDown(metrics.CloudProviderError, metricResourceName, metricGpuType)
```

Comment:
Do you need the extra GPU dimension or is it just to mimic scale up metrics? Similarly for the failure reason - if there is only ever going to be one, is it worth changing `FailedScaleUpReason` type?